docs/reproducibility.md

```diff
@@ -0,0 +1,44 @@
+# Reproducibility
+
+The assumption here is that the original build was done using a Docker build. The
+goal is to reproduce the same MRENCLAVE value. 
+
+You need a couple pieces of information to begin:
+
+- The source MRENCLAVE value, get it from `https://ledgerurl/node/quotes/self`, e.g.:
```

Comment:
Probably not relevant, feel free to ignore: this endpoint would return the MRENCLAVE / quotes data for the specific node in the ledger the client would connect to. If there is more than one node in the network, It is possible that other nodes in the same CCF network have a different MRENCLAVE (e.g., during a code upgrade).

Do you think it could make sense to specify this, or warn users they should check the quotes data for every node if they want to validate all the code versions running in the ledger?

Reply:
No, I think it detracts from the goal of the doc - it is about reproducibility, not about CCF and the way nodes get upgraded